yarn.lock

```diff
@@ -3974,14 +3974,6 @@
     "@typescript-eslint/typescript-estree" "5.3.0"
     debug "^4.3.2"
 
-"@typescript-eslint/scope-manager@5.2.0":
```

Comment:
Dependabot must not have updated these

Reply:
should we remove auto-merge for dependabot updates?

Reply:
No the value that Dependabot adds is worth more than `yarn.lock` being a little inconsistent